meshtastic/__main__.py

```diff
@@ -160,8 +160,8 @@ def setPref(config, comp_name, valStr) -> bool:
     val = meshtastic.util.fromStr(valStr)
     logging.debug(f"valStr:{valStr} val:{val}")
 
-    if snake_name == "psk" and len(valStr) < 8:
-        print(f"Warning: wifi.psk must be 8 or more characters.")
+    if snake_name == "wifi_psk" and len(valStr) < 8:
+        print(f"Warning: network.wifi_psk must be 8 or more characters.")
```

Comment:
This bit hadn't been updated for the new config format, so that's a bug fixed at least.